contracts/token/common/ERC2981/ERC2981Storage.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+
+library ERC2981Storage {
+    struct Layout {
+        // any value between 0 - 10000 should be accepted
+        uint16 royalty;
+        // 1-indexed to allow 0 to signify default royalty
```

Comment:
This comment seems to be unimplemented.  Interesting idea.  Though, it's cheaper to just use the `royalty` value for this purpose.
```suggestion
```